samples/cpp/benchmark_app/main.cpp

```diff
@@ -188,6 +189,22 @@ void setDeviceProperty(ov::Core& core,
     }
 }
 
+void warn_no_batch_for_dynamic(const std::vector<benchmark_app::InputsInfo>& app_inputs_info) {
+    for (const benchmark_app::InputsInfo& inputs : app_inputs_info) {
```

Comment:
I think loop is not needed there. It is enough to check all inputs of first element from app_inputs_info: layouts should be the same

Reply:
Removed